source/onboarding/team/azuread.html.md.erb

```diff
@@ -1,21 +1,24 @@
 ---
 title: Azure AD
-last_reviewed_on: 2021-10-04
+last_reviewed_on: 2022-11-01
 review_in: 6 months
 weight: 1
 ---
 
 # <%= current_page.data.title %>
 
-## Team group
+Azure AD is used to give you and your team-mates access to specific subscriptions in Azure. When your github user has access to a subscription, you will be able to view, create and modify Azure resources for your team.
 
-Create a help request in the [#platops-help](https://hmcts-reform.slack.com/messages/platops-help) channel on Slack.
 
-Description:
+## Access to a an existing team.
 
-> Hi
-> I have just started on the "team name" team,
+Follow the instructions [here](http://127.0.0.1:4567/onboarding/person/#azure-ad-groups)
```

Comment:
avoid using here, it's not very accessible, prefer the page title or a better description, also localhost isn't going to work off your machine

```suggestion
Follow the instructions on the [person onboarding guide](/onboarding/person/#azure-ad-groups).
```